README.md

```diff
@@ -14,7 +14,9 @@ This repository uses submodules for some configuration, so they must be initiali
 ```bash
 $ git submodule init
 $ git submodule update
+$ (cd miniapp/src/main/assets/js-miniapp && git sparse-checkout set js-miniapp-bridge/export/android)
```

Comment:
This should also be added to the CircleCI config (after git submodule update) so that the assembled binaries don't include the extra files.

Reply:
I had updated the CI config.